cli/ClassifyTestCasesCommand.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Yaml;
+
+class ClassifyTestCasesCommand extends Command
```

Comment:
Why no namespace? I'm not a fan of the autoload being "everything could be in `cli`"